docs/planning-experiments/setting_up_the_sample_size_calculator.md

```diff
@@ -0,0 +1,48 @@
+# Setting up the Sample Size Calculator
+
+How long should you plan for a particular experiment to run? Eppo includes tools for estimating in advance when a metric’s Minimum Detectable Effect (MDE) is likely to be achieved. The Minimum Detectable Effect is the smallest lift that can be detected by an experiment a certain percent of the time.
+
+In the past, you may have performed an MDE calculation by manually providing the mean and variance of each metric of interest to a sample-size calculator; Eppo differs from traditional tools with its concept of Entry Points, which computes many of the input values for you. After setting up one or more Entry Points, you can use Eppo’s Sample Size Calculator to quickly and accurately assess the tradeoff between experiment run-time and Minimum Detectable Effect.
+
+## Creating Entry Points
+
+### What is an Entry Point?
+
+Entry Points are Eppo’s way of determining how much traffic a potential experiment will receive.
+
+An Entry Point is a simulated experiment assignment; in other words, it is an event that normally triggers assignment into one or more experiments. Examples of Entry Points might include:
+
+- User logs into website
+- Users enters checkout flow
+- User starts playback
+- Company creates account
+
```

Comment:
Nit: maybe useful to add a screenshot with an entrypoint set up?

Reply:
Added!